cypress/e2e/spoilers.cy.js

```diff
@@ -0,0 +1,19 @@
+import SpoilersPage from '../pages/spoilersPage'
+
+describe('Spoiler Tests', () => {
+  beforeEach(() => {
+    cy.getAllChallenges()
+  })
+
+  it('Check all hints load correctly', () => {
```

Comment:
Can we make this one a little bit more descriptive, like below?
```suggestion
  it('Check all hints load correctly (e.g. have a title and some data)', () => {
```

Reply:
I can make these all a little more verbose if you'd like. I didn't think the assertions themselves were too hard to understand though. 

```
 cy.dataCy(SpoilersPage.SPOILER_TITLE).should('be.visible')
        cy.dataCy(SpoilersPage.SPOILER_TITLE).should('not.be.empty')
        cy.dataCy(SpoilersPage.SPOILER_ANSWER).should('be.visible')
        cy.dataCy(SpoilersPage.SPOILER_ANSWER).should('not.be.empty')
```
Ideally if something is failing i'd like to get the developer reading the test itself as soon as possible.

Reply:
I hang on, I see what ive done. I just copied this test from another page and did not update the title. It is still referring to hints not spoilers.:D

Reply:
Is that better? Or do you think it would still be better more verbose?

Reply:
Okay added this change!